### PR TITLE
Ao

### PR DESCRIPTION
**Pull request by aotian69**
User & SignUp Added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Proposed API:
+- Google Books API: use Google Books API for searching book details.
+
+### More:
+Meetings: every Saturday offline or online meetings
```
Comment (Juntongdalao) [inline README.md:70]: reviewed by Eric

**Review by Juntongdalao — COMMENTED** [review]
reviewed great

**Comment by Juntongdalao** [comment]
reviewed

**Review by Juntongdalao — APPROVED** [review]

**Review by Juntongdalao — APPROVED** [review]
approved

**Review by Juntongdalao — APPROVED** [review]
approved

**Review by lcsway32 — APPROVED** [review]

**Review by lcsway32 — APPROVED** [review]

**Pull request merged**